### PR TITLE
docs: Add design first and design documentation sections, move bash scripts

### LINKED ISSUE #699: Update development process document
**Author the user story for this feature**
Document how to approach development tasks (make sure that design is always a priority and should be always done, references to feature documents we started to create)

### PR DESCRIPTION
**Pull request by illia-romanenko**
Fixes #699

## Checklist

- [x] Documented

### CODE REVIEW

File: docs/02_process.md [hunk 1]
```diff
@@ -28,15 +28,33 @@ Meet industry standards for Agile development using 1 week sprints.
...
+- Analysis
+- Specification
+- Design
+- Development
+- Testing
+- Maintenance
+
+Make sure to commit effort and don't skip steps.
+
+After completing Design and before moving to Development - make sure to approve Design document with your Team Leader or (if you are a Team Leader - with your peer), discuss it with the team.
```
Comment (solid-vovabeloded) [inline docs/02_process.md:44]: Will it be a bit more clear? 
```suggestion
After completing Design and before moving to Development - make sure to approve the Design document with your Team Leader (if you are a Team Leader - with your peer) or discuss it with the team.
```

**Review by solid-vovabeloded — CHANGES_REQUESTED** [review]
Looks great, @illia-romanenko! I've added one minor suggestion here - let me know what do you think.

File: docs/02_process.md (hunk 1 above)
Comment (illia-romanenko) [reply docs/02_process.md]: great thanks!

**Review by solid-vovabeloded — APPROVED** [review]
LGTM, thanks @illia-romanenko! 🚀

**Pull request merged**